hw/top_earlgrey/ip/ast/doc/_index.md

```diff
@@ -1,1060 +1,1178 @@
-
-AST consumes entropy for defensive purposes. However, AST does not
-consume its raw entropy directly. Instead, AST receives entropy from the
-CSRNG. The details of this interface are still under discussion.
-
-
-{{< wavejson >}}
-{
-  signal: [
-    {name: 'clk_ast_es_i' ,  wave: 'p.|..........'},
-    {name: 'entropy_req_o' , wave: '01|.0.1.....0'},
-    {name: 'entropy_ack_i' , wave: '0.|10.1.01..0'},
-    {name: 'entropy_i',      wave: 'xx|2x.22x222x'},
-  ] 
-}
-{{< /wavejson >}}
-
-# Countermeasures and Alerts
-
-## Alert Events
-
-AST’s sensors and detectors, when triggered, output alert events to a
-sensor controller. The event signals are level until acknowledged by the
-controller. Further, the events are differentially encoded to ensure
-they cannot be hard-wired or faulted to either ‘1’ or ‘0’.
-
-Inside the sensor controller, the events are then converted into alerts
-as part of the wider [<u>OpenTitan alert handling
-system</u>](https://github.com/lowRISC/opentitan/blob/master/hw/ip/alert_handler/doc/_index.md).
-
-## Alert Signaling
-
-Outgoing alert events are level. Incoming event ack signals clear the
-alert event (similar to an interrupt). Outgoing alert events should be
-OR'd inside the sensor or power manager (depending on what level of deep
-sleep support is needed) to generate wakeup, that way AST does not need
-to do any additional handling for wakeups during low power mode.
-
-## Countermeasures
-
-Most countermeasure enablement is controlled by Nuvoton via the
-registers interface. Clock jitter is an exception because there is a
-reasoning for dynamically turning it on and off (security/performance
-tradeoff). Unless stated otherwise, countermeasures are active in all
-modes but deep-sleep.
+---
+title: "Analog Sensor Top Technical Specification"
+---
+# Overview
+
+AST, also known as the analog sensor top, is the OpenTitan analog and
+security companion. Within AST are various analog functions (such as
+clocks, regulators, random number generators) needed to make the device
+function, as well as physical security sensors necessary to protect the
+device from physical attacks or manipulation.
+
+At a high level, AST communicates with a number of OpenTitan comportable
+modules. See diagram below.
+
+<img src="./media/image1.png"
+style="width:7.08333in;height:5.60417in" />
+
+In the following sections, each family of connection is briefly
+described and explained. Note, the analog connections to AST are not
+shown in the diagram, but will be explained as well.
+
+# Interface Signals Table
+
+## Table notes
+
+### Signal naming conventions used in this document
+
+It complies with OpenTitan
+[<u>names</u>](https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md#naming)
+and
+[<u>suffixes</u>](https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md#suffixes)
+with some augmentations.
+
+- Clock signals start with clk_*
+
+- Inputs and outputs are marked with *_i/*_o
```

Comment:
*_i/*_o does not get rendered correctly, since `*...*` is a markdown code to make the text italic. you can however escape the asterisks: \*_i/\*_o